docs/content/v0.20/packaging.md

```diff
@@ -10,6 +10,22 @@ There is also a container image involved.
 
 ## KubeStellar
 
+### Derived files
+
+Some files in the kubestellar repo are derived from other files there.
+
+#### Files generated by controller-gen
+
+- `make manifests` generates the CustomeResourceDefinition files.
+
+- `make generated` generates the deep copy code.
+
+Both of the above are currently broken, see Issue 1611.
+
+#### Files generated by code-generator
+
+The files in [pkg/generated](../../../pkg/generated) are generated by [k/code-generator](https://github.com/kubernetes/code-generator). This generation is done at development time by the command `make codegenclients`. That command does only two things: (1) ensure the needed local copy of `code-generator`, and then (2) invoke [hack/update-codegen-clients.sh](../../../hack/update-codegen-clients.sh).
+
```

Comment:
I don't mind having this - but I haven't see someone documenting the fact that they have client-go generated code.

Reply:
I have occasionally seen this documented. It is something that some contributors (the ones who add/change/delete stuff in the API) have to understand, so it should be documented.